playbooks/installation/installation.adoc

```diff
@@ -285,35 +113,22 @@ Overall requirements for Installing OpenShift are very simple:
+NOTE: For more information, see the link:https://docs.openshift.com/enterprise/latest/install_config/install/prerequisites.html#host-preparation[Official Documenation for Host Preparation].
 
-== Ansible Installer
+== Configure and Run the Ansible Installer
 
-We highly recommend using the link:https://docs.openshift.com/enterprise/3.0/admin_guide/install/advanced_install.html#installing-ansible[Advanced Installation method using Ansible] for basically any multi-node installation. The OpenShift Quick Installer is available and useful for quick demos and short-lived installs, but does not support the customization needed to install in many real environments.
+We highly recommend using the link:https://docs.openshift.com/enterprise/latest/install_config/install/advanced_install.html#installing-ansible[Advanced Installation method using Ansible] for basically any multi-node installation. The OpenShift Quick Installer is available and useful for quick demos and short-lived installs, but does not support the customization needed to install in many real environments.
 
 The instructions in the Installer Guide will get you through most basic installs, but there are few additional things to know and be aware of to really understand the installer.
 
-=== Installer Source Code
-
-The Ansible Installer source code is downloaded from link:https://github.com/openshift/openshift-ansible[GitHub]. At the time of writing this document, the docs instruct you to pull down the Master (main) branch of the source code. Changes and bug fixes are merged into this repository on a daily basis, so the installer does not follow the same release cycle that the OpenShift bits do. This means that there are chances that new bug fixes that are merged in could potentially break the installer which had just been working for you the day before. It is important to remember this when you are planning a large environment deployment.
+=== The Ansible Inventory File
 
-IMPORTANT: Our recommendation is to keep a copy of the installer you use to install your environment for re-use when adding nodes or trying to replicate your environment build. When it's time to upgrade to do a new release of the env, you should then pull down the latest installer as a starting point.
-
-=== The Ansible Hosts File
-
-While the Install Guide shows some basic examples for link:https://docs.openshift.com/enterprise/3.0/admin_guide/install/advanced_install.html#configuring-ansible[Configuring Ansible Hosts], there are many more options and variables that can be used to further customize your install. We attempt to break down a few of the common ones here.
+While the Install Guide shows some basic examples for link:https://docs.openshift.com/enterprise/latest/install_config/install/advanced_install.html#configuring-ansible[Configuring Ansible Hosts], there are many more options and variables that can be used to further customize your install. We attempt to break down a few of the common ones here.
```

Comment:
It seems like the official docs have a lot more information than we are providing here. Maybe we should just link to the relevant sections with some opinionated commentary?